xlml/apis/test_config.py

```diff
@@ -101,13 +101,17 @@ class Gpu(Accelerator):
     count: Number of the GPU devices.
     accelerator_type: Type of the accelerator. E.g., `nvidia-test-v100`.
     runtime_version: Runtime image version.
+    network: The network that a GPU will be a part of.
+    subnetwork: The subnetwork that a GPU will be a part of.
   """
 
   machine_type: str
   image_family: str
   count: int
   accelerator_type: str
   runtime_version: Optional[str] = None
+  network: str = 'global/networks/default'
```

Comment:
Are those network and subnetwork used here?

Reply:
Looks `None` and `global/networks/default` are same https://screenshot.googleplex.com/8UxZNGXnwrXLDab - it was `global/networks/default` before so just put it here.

Reply:
`None` makes more sense to me. The network and subnetwork are essentially meaningless if the GPU is in GKE since the network is determined by the cluster itself (not necessarily `default`).